src/index.css

```diff
@@ -0,0 +1 @@
+/* Global Styles */
```

Comment:
either add (if any) or remove.

```
CSSBaseline
```

seems to cover most ground, but maybe we want to set min widths or heights on body etc